tests/testthat/test_array_manipulation.r

```diff
@@ -46,9 +46,17 @@ test_that('get.limit.index sets the right indexes', {
   expect_equal(lim_index$end, c(7, 8, 9, 10))  
 })
 
+
 test_that('ellip.limit.vec returns correctly for numerics', {
-  ellip.h <- '\u22EF'
-  
+  # copy of the function repr_matrix_df.r
+  .char_fallback <- function(char, default) {
```

Comment:
I've no clue, but the tests are anyway failing for me on windows (and I've no clue why). I tried putting the function in once into the test file, but that didn't work either and I don't want to export the function just to make the tests happy...